Owncloud iOs Client/Utils/InfoFileUtils.m

```diff
@@ -228,4 +230,14 @@ + (CustomCellFileAndDirectory *) getTheStatusIconOntheFile: (FileDto *)fileForSe
     return fileCell;
 }
 
+
++(void)createAllFoldersInFileSystemByFileDto:(FileDto *)file andUserDto:(UserDto *)user {
+    
+    NSMutableArray *listOfRemoteFilesAndFolders = [ManageFilesDB getFilesByFileIdForActiveUser:(int) file.idFile];
```

Comment:
Remove (int) cast is not necessary, the method expected a NSInteger and idFile is a NSInteger